lib/OpenQA/Task/Needle/Save.pm

```diff
@@ -151,8 +152,8 @@ sub _save_needle {
     if ($git->enabled) {
         my $error = $git->commit(
             {
-                add     => ["$needledir/$needlename.json", "$needledir/$needlename.png"],
-                message => sprintf("%s for %s", $needlename, $openqa_job->name),
+                add     => ["$needlename.json", "$needlename.png"],
```

Comment:
One question, why needledir is not longer necessary?

Reply:
We're basically using `git -C "$needledir" …` here anyways so it was never necessary. For the tests it was easier to get rid of the full path.